functions/openhab.bash

```diff
@@ -73,7 +73,7 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
 
   if is_pi || is_pine64; then
     cond_echo "Optimizing Java to run on low memory single board computers... "
-    sed -i 's#^EXTRA_JAVA_OPTS=.*#EXTRA_JAVA_OPTS="-Xms250m -Xmx350m"#g' /etc/default/openhab2
+    sed -i 's#^EXTRA_JAVA_OPTS=.*#EXTRA_JAVA_OPTS="-Xms=192m -Xmx320m"#g' /etc/default/openhab2
```

Comment:
not sure what is behind and of the impact on running installations. you can judge better...

Reply:
The =192 is a typo, the values in general should be fine.
I just added an exception for Pi0W.